process/transaction/interceptedTransaction.go

```diff
@@ -262,6 +262,7 @@ func (inTx *InterceptedTransaction) integrity(tx *transaction.Transaction) error
 	if tx.Value.Sign() < 0 {
 		return process.ErrNegativeValue
 	}
+	//TODO: check these checks - why len(RecvUserName) != hasher.Size() ?
```

Comment:
good find - this was changed and forgot it here. user names will be in clear. in the first implementation it was hashed.

Reply:
this changed: can we have only up to 32-byte usernames?

Reply:
yes. changed. only up to 32 bytes

Reply:
ok